cypress/fixtures/dashboard/opensearch_dashboards/data.json.txt

```diff
@@ -0,0 +1,23 @@
+{
+  "type": "doc",
+  "value": {
+    "id": "3",
+    "index": "timestamp-nanos",
+    "source": {
+      "timestamp": "2019-01-01T12:10:30.123456789Z"
```

Comment:
we should make the file name of this not `data.json.txt` since that is too generic of a name and could cause confusion. 

we should also have more realistic data here as in some other fields maybe even number of other timefields. the size should be more than two. this way the test results for casting a wide net of a date range will yield 2 results the same way filtering by the jan 1st of 2019 would which could maybe lead to false positives in our filtering tests